tests/bson/bson-utcdatetime-set_state-002.phpt

```diff
@@ -23,15 +23,15 @@ foreach ($tests as $milliseconds) {
 <?php exit(0); ?>
 --EXPECT--
 MongoDB\BSON\UTCDateTime::__set_state(array(
-   'milliseconds' => 0,
+   'milliseconds' => '0',
```

Comment:
If `var_export()` is always using strings, you should update `bson-utcdatetime-set_state-001.phpt` as well. It no longer needs the EXPECTF and regex patterns to conditionally accept single quotes around these values.